infrastructure/main.tf

```diff
@@ -40,7 +42,7 @@ resource "azurerm_resource_group" "rg" {
 }
 
 resource "azurerm_storage_account" "provider" {
-  name                      = "bulkscanning${var.env}"
+  name                      = "${substr(local.account_name, 0, min(length(local.account_name), 24))}"
```

Comment:
Why was my comment not addressed? We have the same limitation with Vault name and never truncate it, as that could result in name conflicts. We exclude environment name in preview, as the product prefix already provides the environment information.

Reply:
Sorry for that, will do it today.